src/components/join/themes/commerce.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Also, don't believe we need to have themes for these non presentation components.  (join, lazyload, etc.